VERSION

```diff
@@ -0,0 +1 @@
+6.0.7
```

Comment:
What version is this referring to?

Reply:
The WW3 version. I found this version in the release notes and used that.

Reply:
The version number here then is incorrect.  The current version number of WW3 is 7.14 and can be found for example here: https://github.com/NOAA-EMC/WW3/blob/develop/manual/defs.tex#L1 or https://github.com/NOAA-EMC/WW3/blob/develop/model/src/w3initmd.F90#L115

Reply:
Are there no Github releases (or tags) for this version?

Reply:
This is the last tagged/released version: https://github.com/noaa-emc/ww3/tree/6.07.1 there is not a "released" or tagged 7.* version of WW3 yet.